_includes/tutorials/session-windows/ksql/markup/dev/print-topic.adoc

```diff
@@ -0,0 +1,11 @@
+This creates and populate a table that we could query with a `SELECT`, but also a Kafka topic, which we can examine from within KSQL using `PRINT`.
+
++++++
+<pre class="snippet"><code class="shell">{% include_raw tutorials/session-windows/ksql/code/tutorial-steps/dev/print-topic.sql %}</code></pre>
++++++
+
+Notice that the key for each message contains some strange characters that aren't quite printable. KSQL has combined the grouping key (IP address) with its window boundaries using a format that's not quite printable in this format. It should look something like this:
```

Comment:
I didn't see the not-quite-printable effect. Was this an issue in earlier versions, perhaps?
```suggestion
Notice the key for each message. ksqlDB has combined the grouping key (IP address) with its window boundaries. It should look something like this:
```